.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1020 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No. It's one of several that VS keeps creating, it must have slipped into a commit. Will remove.